### PR TITLE
Feature 4: add working-copy and current-version links

### PR DESCRIPTION
**Pull request by justageek**

### CODE REVIEW

**Comment by justageek** [comment]
@e0ipso I'm sure this is likely not the most portable solution at this time.  I just wasn't sure there is a better place to add these links.  The branch only adds the links to the json data.

File: web/modules/contrib/jsonapi/src/LinkManager/LinkManager.php [hunk 1]
```diff
@@ -66,6 +66,34 @@ public function getEntityLink($entity_id, ResourceType $resource_type, array $ro
...
+   *   The JSON API resource type.
+   * @param array $route_parameters
+   *   Parameters for the route generation.
+   * @param string $key
+   *   A key to build the route identifier.
+   *
+   * @return string|null
+   *   The URL string, or NULL if the given entity is not locatable.
+   */
+  public function getEntityVersionLink($entity_id, ResourceType $resource_type, array $route_parameters, $key, $plugin_id) {
```
Comment (e0ipso) [inline web/modules/contrib/jsonapi/src/LinkManager/LinkManager.php]: We are missing documentation for the `$plugin_id` parameter. Have you considered `phpcs` as a pre-push git hook (or maybe even a manual step)?

Comment (q0rban) [reply web/modules/contrib/jsonapi/src/LinkManager/LinkManager.php]: Related: https://github.com/vijaycs85/drupal-quality-checker

Comment (e0ipso) [reply web/modules/contrib/jsonapi/src/LinkManager/LinkManager.php]: Wow, that's pretty great @q0rban!

**Pull request merged**